mara_db/shell.py

```diff
@@ -5,10 +5,11 @@
 """
 
 import shlex
-import sys
 from functools import singledispatch
 
+import sys
 from mara_db import dbs, config
+from mara_db.bigquery import bigquery_credentials
```

Comment:
I think we have a problem here @martin-loetzsch 
When `mara_db.bigquery` is imported in the beginning at the file, it will try to load the module "google.api_core.exceptions" everytime (since it is imported in file `bigquery.py`). But as I understand from the setup.py changes, the google pip packages should be optional.

To fix this, the line
```
from mara_db.bigquery import bigquery_credentials
```
should be removed and in the function `@copy_from_stdin_command.register(dbs.BigQueryDB)` the following line should be added
```
from .bigquery import bigquery_credentials
```
(as it is done already in function `@query_command.register(dbs.BigQueryDB)`